src/main/java/uk/gov/companieshouse/scanupondemand/orders/api/controller/HealthcheckController.java

```diff
@@ -10,8 +10,8 @@
  */
 @RestController
 public class HealthcheckController {
-    @GetMapping("${uk.gov.companieshouse.scanupondemand.orders.api.health}")
-    public ResponseEntity<Void> getHealthCheck (){
-        return ResponseEntity.status(HttpStatus.OK).build();
-    }
+	@GetMapping("${uk.gov.companieshouse.scanupondemand.orders.api.health}")
```

Comment:
Are you using tabs or spaces? I believe the CH convention is 4 spaces for an indentation.

Reply:
My settings were wrong, I corrected them and I reformatted every file. But I will double check if I missed anything